CVs/input4MIPs_source_id.json

```
@@ -89,5 +89,13 @@
         "mip_era":"CMIP6Plus",
         "target_mip":"CMIP",
         "source_version":"4.3"
+    },
+    "UOEXETER-CMIP-0-1-0":{
+        "contact":"t.aubry@exeter.ac.uk",
+        "further_info_url":"http://www.tbd.invalid",
+        "institution_id":"uoexeter",
+        "license_id":"CC BY 4.0",
+        "mip_era":"CMIP6Plus",
+        "source_version":"0.1.0"
     }
```

@thomasaubry just triple checking I copied this across correctly?

---

@znichollscr should it be 1-1-0 and 1.1.0? I've been pulling hair for a couple hours trying to upload my emission file on the ftp and wondering if it's related. It's now consistently 1-1-0 in my notebook and source_id file.

---

Perhaps. It's totally up to you. The change below is needed to be consistent with your notebook right?

```suggestion
    "UOEXETER-CMIP-1-1-0":{
        "contact":"t.aubry@exeter.ac.uk",
        "further_info_url":"http://www.tbd.invalid",
        "institution_id":"uoexeter",
        "license_id":"CC BY 4.0",
        "mip_era":"CMIP6Plus",
        "source_version":"1.1.0"
    }
```

---

Perhaps. It's totally up to you. The change below is needed to be consistent with your notebook right?

```suggestion
    "UOEXETER-CMIP-1-1-0":{
        "contact":"t.aubry@exeter.ac.uk",
        "further_info_url":"http://www.tbd.invalid",
        "institution_id":"uoexeter",
        "license_id":"CC BY 4.0",
        "mip_era":"CMIP6Plus",
        "source_version":"1.1.0"
    }
```

---

@thomasaubry ping here